runtime/binding-mqtt-kafka/src/main/java/io/aklivity/zilla/runtime/binding/mqtt/kafka/internal/stream/MqttKafkaProxyFactory.java

```diff
@@ -55,7 +56,7 @@ public MqttKafkaProxyFactory(
             config, context, bindings::get);
 
         final MqttKafkaSubscribeFactory subscribeFactory = new MqttKafkaSubscribeFactory(
-            config, context, bindings::get);
+            config, context, bindings::get, BOOTSTRAP_STREAM_RECONNECT_DELAY);
```

Comment:
Suggest calling `config.bootstrapStreamReconnectDelay()` from constructor instead of passing property to constructor.

Equivalent feedback for `MqttKafkaSessionFactory` and `WILL_STREAM_RECONNECT_DELAY` property.

Reply:
This feedback for `BOOTSTRAP_STREAM_RECONNECT_DELAY` has not yet been applied.